examples/pytorch/audio-classification/run_audio_classification.py

```diff
@@ -382,18 +385,32 @@ def compute_metrics(eval_pred):
         tokenizer=feature_extractor,
     )
 
+    # mlflow initial
+    experiment_name ='translateion-{}'.format(model_args.model_name_or_path)
+    current_experiment = mlflow.set_experiment(experiment_name)
+    experiment_id=current_experiment.experiment_id
+    mlflow_runner = mlflow.start_run(run_name=model_args.model_name_or_path, experiment_id=experiment_id)
```

Comment:
Model Runner sets up all the run_id and run_name, experiment arguments.
Just call mlflow.start_run().